content/content-strategy/define-user-content-needs/create-actionable-artefacts/journey-maps/supporting-takeaway.md

```diff
@@ -0,0 +1,4 @@
+---
+layout: text/callout
+---
+### [1]Prep time: 2 hrs   People: 2 to 10   Time to run: 2 hrs to 1 day
```

Comment:
Ignoring format for now, will be replaced with new partial. Heading and display style are incompatible for this context.

Reply:
Steve fixed